### ISSUE TITLE
How to represent explicit modeled connectors?

### ISSUE THREAD

**Issue by oalt** [GfSE/SysML-SpecIF-Bridge#8]
To avoid defining a special statement for each connector type, I used in my prototype implementation the dcterms:type property and a very general Statement class (UML:Relationship).

You, @odungern have defined different types of statements to map SysML connectors in your SpecIF example (Small Vehicle). 

How should we proceed in this case? Define a lot of special statement classes or use one and use a type property?

**Comment by odungern** [comment]
You are right. In case of resources we try to reduce the number of classes and use subClass properties to distinguish different flavors. The criterion whether resources get the same resourceClass is whether they can be included in the same set of statements (by means of subjectClasses and objectClasses) or not.
Applying the same thinking to statementClasses: all statement types which have the same set of subjectClasses and objectClasses shall be combined in a common statementClass. For example 'is associated with', 'aggregates' and 'is composed of' would be 3 statement types that can be combined in a single statementClass with different 'SpecIF:SubClass' values.
Wrt to using 'rdf:type' or 'SpecIF:SubClass': I have done some research on how 'rdf:type' is being used. All I have found is a relationship between type/class and instance. So my conclusion is that 'rdf:type' should be used to characterize such relationships. 'SpecIF:SubClass' is not a relationship type, but a property that assigns a finer grained type to a resource instance or a statement instance ... in addition to it's 'class'. Do you agree?

**Comment by odungern** [comment]
@oalt: Today we have discussed whether we can use dcterms:type rather than SpecIF:SubClass. 
I have found the definition of [dcterms:type](https://terms.tdwg.org/wiki/dcterms:type) and I think it fits well. So I propose to use it instead of SpecIF:SubClass.
Also I think that the [DCMI Type Vocabulary](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#section-7) defines a fair number of language-independent values which we can use for model integration.
